src/main/java/io/cloudsoft/winrm4j/winrm/WinRmTool.java

```diff
@@ -24,13 +24,49 @@ public static WinRmTool connect(String address, String username, String password
         return tool;
     }
 
+    /**
+     * Execute a list of Windows Native commands as one command.
+     * The method translates the list of commands to a single String command with a <code>"\r\n"</code> delimiter and a terminating one.
+     * @param commands
+     * @deprecated since 0.2; Use the {@link #executeCommand(String)} instead and transform your commands list explicitly
+     */
+    @Deprecated
     public WinRmToolResponse executeScript(List<String> commands) {
         Response response = session.run_cmd(compileScript(commands));
         return responseToWinRmToolResponse(response);
     }
 
-    public WinRmToolResponse executePs(List<String> commands) {
-        Response response = session.run_ps(compileScript(commands));
+    /**
+     * Executes a Native Windows command.
+     * It is creating a new Shell on the destination host each time it is being called.
+     * @param command
+     * @since 0.2
+     */
+    public WinRmToolResponse executeCommand(String command) {
```

Comment:
Can you explain the reason for the method renames (rather than just overloading the existing methods that have the same behaviour - e.g. `executeScript(List)` is exactly like this `executeCommand(String)` if you pass a list of size one).